docs/introduction.md

```diff
@@ -53,23 +54,22 @@ documentation should of course be the first step. But where do you go
 after that?
 
 
-### Understanding Dataflow Architecture
+### Understanding the Reactive Programming Model
 
 The key to writing good, performant deck.gl layers lies in understanding
-how to minimize updates of any calculated data, such as WebGL.
+how to minimize updates of any redundant calculations, such as WebGL.
```

Comment:
missed this, was not sure how to fix the such as part, thanks for the suggestion.